### DISCUSSION TITLE
Gitactions workflow triggered with repository_dispatch always has the status 'Completed'

### DISCUSSION THREAD

**Discussion by isurulucky** [community/community#27134] [Actions]
<p>Hi,</p>
<p>Noted $subject today, a few hours ago. See the attached screenshot; even though the gitactions workflow is ongoing, the status is always <code>completed</code>. We have an integration written around gitactions, invoked via the repository dispatch event. It uses the <code>in_progress</code> state of workflow runs to filter out the currently executing ones, and its failing due to this problem. However, when you re-run using the browser, the state changes correctly (queued -&gt; in progress -&gt; completed).</p>
<p>Is this a known issue? How can this be fixed? Or am I missing something?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/b/9b234bff6dc2be0672d30b82f611a054e2fe6b91.png" data-download-href="/uploads/short-url/m8pD7YqNiO9jw61IsPusI27hva1.png?dl=1" title="gitactions_completed" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/2777052/181102008-22188843-14a9-49df-8aa7-1772f06064e3.png" alt="gitactions_completed" data-base62-sha1="m8pD7YqNiO9jw61IsPusI27hva1" width="690" height="302" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9b234bff6dc2be0672d30b82f611a054e2fe6b91_2_690x302.png, https://aws1.discourse-cdn.com/github/original/3X/9/b/9b234bff6dc2be0672d30b82f611a054e2fe6b91.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/b/9b234bff6dc2be0672d30b82f611a054e2fe6b91.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9b234bff6dc2be0672d30b82f611a054e2fe6b91_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">gitactions_completed</span><span class="informations">1025×449 43.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by AntonPuko** [comment]
<p>same here, looks like some bug on the github side</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Yep, looks like there’s been an issue with reporting the correct status of workflow runs, sorry about that!</p>
<p>The engineers are aware of it and are looking into it. I believe this may already actually be fixed for <em>future</em> workflow runs, so you shouldn’t see this happen with any new ones (although please post if it is still happening), but any runs that happened during that time may still report an incorrect status for the time being. This <em>should</em> fix itself for these runs when the runs <em>actually do complete</em> though.</p>
<p>If there’s any future updates I’ll keep you all informed too :v:</p>
